src/SystemCommands-RefactoringSupport/RBAbstractSelectionPresenter.class.st

```diff
@@ -0,0 +1,72 @@
+Class {
+	#name : #RBAbstractSelectionPresenter,
+	#superclass : #RBItemsSelectionPresenter,
+	#instVars : [
+		'dropList',
+		'dropLabel',
+		'acceptBlock'
+	],
+	#category : #'SystemCommands-RefactoringSupport'
+}
+
+{ #category : #specs }
+RBAbstractSelectionPresenter class >> defaultSpec [
+	^ SpBoxLayout newVertical
+		add: #dropLabel withConstraints: [ :c | c height: self labelHeight ];
+		add:
+			(SpBoxLayout newHorizontal
+				add: #dropList ;
+				yourself) withConstraints: [ :c | c height: self toolbarHeight  ];
+		add: #label withConstraints: [ :c | c height: self buttonHeight ];
+		add:
+			(SpBoxLayout newHorizontal
```

Comment:
Can you use leftToRight instead of newHorizontal?
Because newHorizontal sucks!
Check the implementation to make sure that I'm right
newHorizontal and newVertical are bad names.